dog_breed_identification/predict_model.py

```diff
@@ -2,10 +2,13 @@
 import torchvision
 import os
 import pandas as pd
+import click
 from dog_breed_identification.models.model import Model
 
-
-def predict(model: torch.nn.Module, directory: str, return_label: bool = True) -> None:
+@click.command()
+@click.argument("model_params")
+@click.argument("directory")
+def predict(model_params: torch.nn.Module, directory: str, return_label: bool = True) -> None:
```

Comment:
Pls import Module from torch.nn for better readibility.

Reply:
fixed